x-pack/plugins/rule_registry/server/routes/find.ts

```diff
@@ -27,8 +27,8 @@ export const findAlertsByQueryRoute = (router: IRouter<RacRequestHandlerContext>
               index: t.string,
               query: t.object,
               aggs: t.union([
-                t.record(t.string, bucketAggsSchemas),
                 t.record(t.string, metricsAggsSchemas),
+                t.record(t.string, bucketAggsSchemas),
```

Comment:
@XavierM, I changed the order as `metricsAggsSchemas` is more strict than `bucketAggsSchemas`, so `io-ts` will check it first in the `union` function. Besides, this update fixes the failing test!